crates/autopilot/src/run_loop.rs

```diff
@@ -1128,20 +1126,14 @@ pub mod observe {
     }
 
     /// Records metrics for the matched but unsettled orders.
-    pub fn unsettled(solutions: &[domain::competition::Participant], auction: &domain::Auction) {
-        let Some(winner) = solutions.first() else {
-            // no solutions means nothing to report
-            return;
-        };
-
+    pub fn unsettled(ranking: &Ranking, auction: &domain::Auction) {
         let mut non_winning_orders = {
```

Comment:
would simple ranking.non_winners() work here?

Reply:
Do you mean collecting the orders from all `non_winners()` instead of what we do now? That would not work because a non-winner might suggest to settle an order which is actually getting settled by a winning solver. Here we only want to track how many solutions are currently solvable that aren't getting solved.